lib/cookies.js

```diff
@@ -10,43 +10,30 @@
  * See the License for the specific language governing permissions and limitations under the License.
  *
  */
-
-var util = require('./util');
+var Cookies = require('js-cookie');
 
 function setCookie(name, value, expiresAt) {
-  var expiresText = '';
-  if (expiresAt) {
-    expiresText = ' expires=' + util.isoToUTCString(expiresAt) + ';';
+  var expires = expiresAt || new Date('2038-01-19T03:14:07.000Z');
+
+  if (typeof expires === 'string') {
+    // Possible that the expiresAt value is passed via JSON
+    expires = new Date(expires);
   }
```

Comment:
Unfortunately, the `js-cookie` library's `Cookies.set` method only accepts:
  -  `Number`: Number of days the cookie will be active for
  -  `Date`: Date object

The way I am thinking about this is as follows:
- We're going to proxy the `set` method with default values, given a `key: value` pair and a expiry time
- Since this is an internal method, we can assume/assert that the `expiresAt` param is a string

However, we should probably catch these errors- so I'll make sure to update this method accordingly!
👍